introduce a new set APIs to create/update/delete page sections (#19844)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/verticalsection-delete.md

---
title: 'Delete a verticalSection'
description: 'Delete a vertical section from a page.'
author: sangle7
ms.localizationpriority: medium
ms.prod: sharepoint
doc_type: apiPageType
---

> # Delete a vertical section from a page

Reviewer: Can we leave this as "Delete verticalSection"? Or is there a reason we need to specify that we are deleting a vertical from a page?

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Removes a [verticalSection][] from the [sitePage][]

Reviewer: Please add the links of the resources.

> [verticalSection]: ../resources/verticalSection.md

Reviewer: Please insert the links in line 16 accordingly.

> [sitePage]: ../resources/sitepage.md

Reviewer: Please insert the link in line 16 accordingly.

## Permissions

One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

>**Note:** To delete an item, the user must have granted the application write access to the item to be deleted.

|Permission type      | Permissions (from least to most privileged)              |
|:--------------------|:---------------------------------------------------------|
|Delegated (work or school account) | Sites.ReadWrite.All    |
|Delegated (personal Microsoft account) | Not supported.    |
|Application | Sites.ReadWrite.All |

## HTTP request

<!-- { "blockType": "ignored" } -->

```http
DELETE /sites/{site-id}/pages/{page-id}/canvasLayout/verticalSection
```

> 

Reviewer: Please confirm there isn't any info to add in the ## Request headers section.

Author: Updated

## Request body

Do not supply a request body with this method.

## Response

> If successful, this call returns a `204 No Content` response to indicate that the resource was deleted and there was nothing to return.

Reviewer: ```suggestion
If successful, this call returns a `204 No Content`. It does not return anything in the response body.
```

---
title: 'Delete a verticalSection'
description: 'Delete a vertical section from a page.'
author: sangle7
ms.localizationpriority: medium
ms.prod: sharepoint
doc_type: apiPageType
---

> # Delete vertical section from a page

Reviewer 2: Please update the title and H1 to: "Delete verticalSection".

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Removes a [verticalSection](../resources/verticalSection.md) from the [sitePage](../resources/sitepage.md).

Reviewer 2: Revise as:
Remove a [verticalSection](../resources/verticalsection.md) from a [sitePage](../resources/sitepage.md).

>**Note:** To delete an item, the user must have granted the application write access to the item to be deleted.

|Permission type      | Permissions (from least to most privileged)              |
|:--------------------|:---------------------------------------------------------|
|Delegated (work or school account) | Sites.ReadWrite.All    |
|Delegated (personal Microsoft account) | Not supported.    |
|Application | Sites.ReadWrite.All |

## HTTP request

<!-- { "blockType": "ignored" } -->

```http
DELETE /sites/{site-id}/pages/{page-id}/canvasLayout/verticalSection
```

## Request headers

| Name      |Description|
|:----------|:----------|
| Authorization | Bearer \{token\}. Required. |

## Request body

Do not supply a request body with this method.

## Response

If successful, this call returns a `204 No Content`. It does not return anything in the response body.

If the vertical section is not found, this method returns `404 Not Found`.

## Example

<!-- { "blockType": "request", "name": "delete-verticalSection", "scopes": "sites.readwrite.all" } -->

> ##### Request

Reviewer 2: Changes these to H3s.
